modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java

```diff
@@ -1935,41 +1935,61 @@ private boolean dumpLongRunningOperations0(long timeout) {
         final IgniteDiagnosticPrepareContext diagCtx = cctx.kernalContext().cluster().diagnosticEnabled() ?
             new IgniteDiagnosticPrepareContext(cctx.localNodeId()) : null;
 
+        WarningsGroup warnings = new WarningsGroup(diagnosticLog, 10);
```

Comment:
Can we please extract this number to IGNITE_foo system property?

Reply:
Yes, this makes sense. However other parts of this class don't make a similar parameter configurable.

https://github.com/apache/ignite/blob/26a15264ae69e4127f9e0b3e88830bf3edc56ece/modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java#L1859

https://github.com/apache/ignite/blob/26a15264ae69e4127f9e0b3e88830bf3edc56ece/modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java#L2729

Reply:
I would propose to introduce a single sys var to configure all three of them. Use default values (5, 10) if it is not present.

Reply:
Done